architecture/launch/sitl_laptop_collectData.launch

```diff
@@ -24,60 +24,38 @@
         <rosparam command="load" file="$(find architecture)/cfg/sim_hitl_params.yaml" />
-        <param name="target_frame"   value="base_link" />
+        <param name="target_frame" value="base_link" />
         <remap from="cloud_filtered" to="free_cloud" />
         <remap from="scan" to="hokuyo_laser_scan" />
     </node>
     <!-- #######################  Octomap  ######################################## -->
     <node pkg="octomap_server" type="octomap_server_node" name="octomap_builder" output="screen">
-        <param name="frame_id" type="string"    value="map" />   
+        <param name="frame_id" type="string" value="map" />
         <remap from="cloud_in" to="/free_cloud" />
-        <param name="latch"                     value="false" />
-        <param name="publish_free_space"        value="true"/>
-        <param name="sensor_model/max_range"    value="50"/>
-        
-        <param name="sensor_model/hit"          value="0.74"/>
-        <param name="sensor_model/miss"         value="0.48464"/>
-        
-        <param name="resolution"                value="1" />
+        <param name="latch" value="false" />
+        <param name="publish_free_space" value="true"/>
+        <param name="sensor_model/max_range" value="50"/>
+        <param name="sensor_model/hit" value="0.74"/>
+        <param name="sensor_model/miss" value="0.48464"/>
+        <param name="resolution" value="1" />
     </node>
-    <!-- #######################  PX4  ############################################ -->
-    <node name="sitl" pkg="px4" type="px4" output="screen"
-        args="$(find px4) $(arg rcS)">
-    </node>
-    <node name="mav_comms"          type="mav_comms_node"       pkg="architecture"  output="screen" />
     <!-- #######################  Architecture nodes  ############################# -->
-    <node name="lazy_theta_star"    type="ltStar_async_node"    pkg="lazy_theta_star" output="screen" /> 
-    <node name="state_manager"      type="state_manager_node"   pkg="architecture"  output="screen" />
-    <!-- <node name="frontier_cells"     type="frontiers_async_node" pkg="frontiers"     output="screen" /> --> 
-    <node name="middle_man_current_position" type="current_position_provider_node" pkg="architecture"  output="screen" />
- 
+    <!-- <node name="lazy_theta_star" type="ltStar_async_node" pkg="lazy_theta_star" output="screen" /> -->
+    <!-- <node name="state_manager" type="state_manager_node" pkg="architecture" output="screen" /> -->
+    <!-- <node name="frontier_cells" type="frontiers_async_node" pkg="frontiers" output="screen" /> -->
+    <node name="middle_man_current_position" type="current_position_provider_node" pkg="architecture" output="screen" />
+    <!-- #######################  Save Poses Node  ############################# -->
+    <node name="save_poses" type="save_poses_node" pkg="architecture" output="screen" required="true"/>
```

Comment:
What is this node save_poses_nodes?
I'm not seeing any new python or cpp files.
Not sure if there is a file missing or the line shouldn't be there.
Was this to record the information in the pdf?

Reply:
```suggestion
    <node name="save_poses" type="save_poses_node" pkg="architecture" output="screen" required="true"/>
```

This line should not be there.